### PR TITLE
ITS ALIVE!!!!

### PR DESCRIPTION
**Pull request by ellenkorbes**
Because #10 & #15.

This is Jekyll (https://jekyllrb.com/docs/). 

Just `make` as usual and go to `http://localhost:3000/`. Changes to the `jekyll/` folder (e.g. `jekyll/_posts/` stuff) regenerate the `public/` pages automatically.

### CODE REVIEW

**Comment by kris-nova** [comment]
Can we use port 80 instead of 3000 please?

**Comment by ellenkorbes** [comment]
> Can we have the port defined in the Makefile

I tried `export PORT = 80` on Makefile and `EXPOSE $PORT` or `EXPOSE $(PORT)` on Dockerfile; a million different variations, etc. 

Couldn't make it work for now.

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:boron-alpine
+FROM starefossen/ruby-node
```
Comment (kris-nova) [inline Dockerfile]: We are using ruby?

Comment (ellenkorbes) [inline Dockerfile]: `jekyll build` <- ruby.

**Pull request merged**